client/components/deposit-status-pill/test/index.test.tsx

```diff
@@ -10,12 +10,6 @@ import { render } from '@testing-library/react';
 import DepositStatusPill from '..';
 
 describe( 'Deposits status pill renders', () => {
-	test( 'Renders default status pill "unknown" when unknown/invalid status is given.', () => {
-		const { getByText } = render( <DepositStatusPill status="hello" /> );
-
-		expect( getByText( 'Unknown' ) ).toBeTruthy();
-	} );
-
```

Comment:
I had to get rid of this test because it checked what would happen if you tried to create a deposit pill component with a fake status. Given this prop is typed now, it's no longer possible.